### DISCUSSION TITLE
How to try FreshRSS easily? Demo pages

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#3770] [Ideas]
Hey,

I found another way to try out FreshRSS next to demo.freshrss.org (that does not work in these days (see #3757):

FreshRSS is listed on Softaculous: https://www.softaculous.com/apps/rss/FreshRSS
So it is possible to create a fresh online demo installation:
![grafik](https://user-images.githubusercontent.com/1645099/129479982-31b33775-ea00-4774-b86b-9057e71752bd.png)

After some seconds it is installed there.

If the login (user/Pw = admin / password) does not work:
open the demo in a new tab via: right click on the FreshRSS logo and choose: open in a new tab. Then FreshRSS should be usable without the login information header of Softaculous (reason: your browser will block it because of " SecurityError: Permission denied to access property "document" on cross-origin object").

(Windows operation system:) Another nice option is to download Softaculous AMPPS (local PHP+Database+Webserver) and install FreshRSS on your localhost.

**Comment by Alkarex** [comment]
Nice to know :-)

**Comment by math-GH** [comment]
Would it make sense to include this information into the documentation or just leave it here in the forum?

**Reply by Alkarex** [reply]
If you find a nice place in the documentation, please do :-)
